shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/CompiledGenerator.java

```diff
@@ -127,9 +128,13 @@ public void errorHtml(SectionRenderer renderer) {
 	public <T> void run(ActionConsumer consumer, Function<Class<T>, Stream<T>> input) {
 		// Load all the input data in an attempt to cache it before any olives try to
 		// use it. This avoids making the first olive seem really slow.
-		InputFormatDefinition.formats()
-				.forEach(format -> INPUT_RECORDS.labels(format.name()).set(format.input(format.itemClass()).count()));
-		ActionGenerator.OLIVE_FLOW.clear();
+		InputFormatDefinition.formats().forEach(format -> {
+			try {
+				INPUT_RECORDS.labels(format.name()).set(format.input(format.itemClass()).count());
+			} catch (Exception e) {
+				e.printStackTrace();
```

Comment:
Is it a good idea to printStackTrace here? does Shesmu have a logger library yet?

Reply:
Yes. If there's a broken input source, that error should go to the console.

There's no logger and given it runs as a standalone application, I'm not sure there's a reason to have a logger. Everything ends up in the system log if it goes to the console.

Reply:
:+1: